src/types/index.d.ts

```diff
@@ -298,3 +298,23 @@ export type FilterExpressionRange = {
 };
 
 export type FilterExpressionRangeValue = ['-inf' | number, 'inf' | number];
+
+export interface SearchabilityConfigurationResponse {
+  name: string;
+  fuzzy_searchable: boolean,
+  exact_searchable: boolean,
+  type: string,
+  displayable: boolean,
+  hidden: boolean,
+  created_at: string,
+  updated_at: string
+}
+
+export interface SearchabilityConfiguration {
```

Comment:
Are all the fields here required?

Reply:
No I copied from the response one and forgot to add optional. Will update, thank you!

Reply:
Done here: [ae21f69](https://github.com/Constructor-io/constructorio-node/pull/155/commits/ae21f69a25c026e34e5827a7355e659ccc0f5609)